### DISCUSSION TITLE
1.3.0-beta1 release was not created on GitHub

### DISCUSSION THREAD

**Discussion by DX37** [d4rken-org/sdmaid-se#1482] [General]
**Describe the bug**
There wasn't `1.3.0-beta1` release created on repository releases (despite having tag), but this version was uploaded to Google Play. I can't update to it because of direct updates from repo with Obtanium. I guess it's because of failed job, which has just failed without any useful info https://github.com/d4rken-org/sdmaid-se/actions/runs/11738234877/job/32700367885

**Comment by NurMiApp** [comment]
I'm sorry Matthias was bad. We are going back SD Maid the triage bugs where is it located.

Regards, the Nurkhametov Mike.

**Comment by d4rken** [comment]
V1.3.0-beta1 has no functional changes, it only fixed the Google play release pipeline.

Beta0 was blocked due to unsupported languages

https://github.com/d4rken-org/sdmaid-se/commit/9bbcaad86b36c49f9ab8aece7b61aea17fdd8034

**Reply by DX37** [reply]
> V1.3.0-beta1 has no functional changes, it only fixed the Google play release pipeline.
> 
> Beta0 was blocked due to unsupported languages
> 
> [9bbcaad](https://github.com/d4rken-org/sdmaid-se/commit/9bbcaad86b36c49f9ab8aece7b61aea17fdd8034)

Got it. Thanks for reply.
